code/datums/components/footstep.dm

```diff
@@ -26,10 +26,10 @@
 				return COMPONENT_INCOMPATIBLE
 			RegisterSignal(parent, COMSIG_MOVABLE_MOVED, .proc/play_humanstep)
 			return
-		if(FOOTSTEP_MOB_CLAW)
-			footstep_sounds = GLOB.clawfootstep
 		if(FOOTSTEP_MOB_BAREFOOT)
 			footstep_sounds = GLOB.barefootstep
+		if(FOOTSTEP_MOB_CLAW)
+			footstep_sounds = GLOB.clawfootstep
```

Comment:
I'm... not entirely sure to be honest. I think it was to make it more consistent with the order of the lists defined in [code/__DEFINES/footstep.dm](https://github.com/ParadiseSS13/Paradise/blob/master/code/__DEFINES/footstep.dm), and also because Barefoot>Claw>Heavy makes more sense to me in terms of readability, although it's not a massive difference.